src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -59,7 +60,21 @@ const renderExpandedView = (row) => {
                             {suggestedInstances}</Td>
                     </Tr>
                 </Tbody>
-            </TableComposable> }
+            </TableComposable>
+            }
+            {
+                psiEnabled &&
```

Comment:
@r14chandra I think you missed put ! here as we want to show the message for systems where PSI is not enabled